ddl/db_integration_test.go

```diff
@@ -2126,6 +2124,9 @@ func (s *testIntegrationSuite7) TestAddExpressionIndex(c *C) {
 	tk.MustExec("alter table t1 add unique index ei_ab ((concat(a, b)));")
 	tk.MustExec("alter table t1 alter index ei_ab invisible;")
 
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t(a int, key((a+1)), key((a+2)), key idx((a+3)), key((a+4)));")
```

Comment:
Do we need to check the result of `show create table` to ensure the correctness of `setEmptyConstraintName`?

Reply:
I think we can move this check to the QA stage, then we can use mysql_test to check it. UT is not suited for checking `show create table`'s result.